packages/autorest.typescript/src/transforms/operationTransforms.ts

```diff
@@ -607,3 +610,21 @@ function getMapperForSchema(
     })
   );
 }
+
+function isOperationTenantLevel(operation: Operation) {
```

Comment:
Is my understanding correct?

If one operation has method-level subid defined, the method would return `true`?